src/App.js

```diff
@@ -16,7 +16,7 @@ class Chart extends Component {
   }
 
   render() {
-    return <div ref="chart" />;
+    return <div ref={el => this.chartEl = el} />;
```

Comment:
here's your issue - you need a "callback ref" to access the underlying DOM element - "https://facebook.github.io/react/docs/refs-and-the-dom.html#adding-a-ref-to-a-dom-element"

Reply:
Interesting, thanks!